src/ui/components/ExplorerHome/ExplorerHome.js

```diff
@@ -190,6 +215,13 @@ const TIMEFRAME_OPTIONS: Array<{|
             startTimeMs: intervals[0].startTimeMs,
             endTimeMs: intervals[intervals.length - 1].endTimeMs,
           },
+    chartSelector: (intervals) =>
+      intervals.length === 1
+        ? intervals[0]
+        : {
+            startTimeMs: intervals[0].startTimeMs,
+            endTimeMs: intervals[intervals.length - 1].endTimeMs,
+          },
```

Comment:
Just noticed some redundancy in this code I wrote 😯
```suggestion
          {
            startTimeMs: intervals[0].startTimeMs,
            endTimeMs: intervals[intervals.length - 1].endTimeMs,
          },
```

Reply:
Excellent! Wouldn't the same be true for lines 212-224?

Reply:
correct